### PR TITLE
New package: libmypaint-1.3.0

### PR DESCRIPTION
**Pull request by kartik-ynwa**
New package: mypaint-brushes-1.3.0

Dependencies for GIMP 2.10.0

### CODE REVIEW

File: srcpkgs/libmypaint/template [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+version=1.3.0
+revision=1
+build_style=gnu-configure
+configure_args="--disable-gegl"
+hostmakedepends="intltool pkg-config"
+makedepends="json-c-devel gobject-introspection glib-devel json-glib-devel"
+short_desc="MyPaint brush engine library"
+maintainer="Kartik S <kartik.ynwa@gmail.com>"
+homepage="https://github.com/mypaint/${pkgname}"
+license="ISC"
```
Comment (maxice8) [inline srcpkgs/libmypaint/template]: move license= to above homepage=

Comment (kartik-ynwa) [reply srcpkgs/libmypaint/template]: Done.

**Comment by kartik-ynwa** [comment]
We did it!

File: srcpkgs/libmypaint/template [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+version=1.3.0
+revision=1
+build_style=gnu-configure
+configure_args="--disable-gegl"
+hostmakedepends="intltool pkg-config"
+makedepends="json-c-devel gobject-introspection glib-devel json-glib-devel"
+short_desc="MyPaint brush engine library"
+maintainer="Kartik S <kartik.ynwa@gmail.com>"
+license="ISC"
+homepage="https://github.com/mypaint/${pkgname}"
```
Comment (cr6git) [inline srcpkgs/libmypaint/template]:
````
homepage="https://github.com/mypaint/libmypaint"
distfiles="https://github.com/mypaint/libmypaint/releases/download/v${version}/libmypaint-${version}.tar.xz"
````

File: srcpkgs/mypaint-brushes/template [hunk 3]
```diff
@@ -0,0 +1,21 @@
+# Template file for 'mypaint-brushes'
+pkgname=mypaint-brushes
+version=1.3.0
+revision=1
+build_style=gnu-configure
+hostmakedepends="automake"
+makedepends="libmypaint-devel"
+short_desc="Brushes used by MyPaint and other software using libmypaint"
+maintainer="Kartik Singh <kartik.ynwa@gmail.com>"
+license="CC0-1.0, GPL-2.0-or-later, GPL-3.0-only"
```
Comment (cr6git) [inline srcpkgs/mypaint-brushes/template]: `GPL-3.0-or-later` instead of `GPL-3.0-only`.

File: srcpkgs/mypaint-brushes/template [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+homepage="https://github.com/Jehan/mypaint-brushes/"
```
Comment (cr6git) [inline srcpkgs/mypaint-brushes/template]:
````
homepage="https://github.com/Jehan/mypaint-brushes"
distfiles="https://github.com/Jehan/mypaint-brushes/archive/v${version}.tar.gz"
````

**Comment by kartik-ynwa** [comment]
@cr6git  Done

**Comment by kartik-ynwa** [comment]
No worries my dude

**Pull request merged**